mkdocs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+mkdocs==1.6.1
```

Comment:
Are mkdocs releases variable enough that this is needed over just `pip install --no-cache-dir mkdocs` as mentioned in their docs?

Reply:
Sorry i don't understand exactly what you mean and i couldn't find reference to the mkdocs "release variable"?

You could install with `pip install --no-cache-dir mkdocs mkdocs-meterial` or you could install using the requirements file:  `pip install -r requirements.txt`.  
I just realized that some of the requirements I had were not needed and I've just simplified it to:
```
mkdocs==1.6.1
mkdocs-material==9.5.49
```
mkdocs-material is the theme being used. I tried it without and readability is objectively worse with default mkdocs theme, which is probably why the material theme seems to be a ubiquitous choice for many mkdocs sites.